Automattic/pocket-casts-android
# Sync use embedded artwork setting

## Description

One of the settings listed for sync project #1739. This PR adds syncing of the global settings for displaying embedded artwork.

## Testing Instructions

I'm having trouble testing this through the app behavior because of the whole embedded/episode artworks. If you know how to test it properly I'd be happy to know.

1. Have two devices D1 and D2.
2. Make sure that both devices are synced before starting the test by refreshing apps in the profile.
3. D1: Go to `Profile`/`Settings (cog icon)`/`Appearance`.
4. D1: Toggle `Use embedded podcast artwork` to `ON`.
5. D1: Sync the device in the `Profile`.
6. D2: Sync the device in the `Profile`.
7. D2: Go to `Profile`/`Settings (cog icon)`/`Appearance`.
8. D2: See if `Use embedded podcast artwork` is synced to `ON`.
9. D2: Toggle `Use embedded podcast artwork` to `OFF`.
10. D2: Sync the device in the `Profile`.
11. D1: Sync the device in the `Profile`.
12. D1: Go to `Profile`/`Settings (cog icon)`/`Appearance`.
13. D1: See if `Use embedded podcast artwork` is synced to `ON`.

## Checklist
 
#### I have tested any UI changes...
